src/app/services/movies/movies.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { map, catchError } from 'rxjs/operators';
+import { throwError } from 'rxjs';
+
+const tmdbKey: string = '06f99320c4d4aafa43383b3d6c8da151';
+const imageURL: string = 'https://image.tmdb.org/t/p/w500';
+
+const getLink = (category: string): string =>
+  `https://api.themoviedb.org/3/movie/${category}?api_key=${tmdbKey}`;
+
+@Injectable({
+  providedIn: 'root',
+})
+export class MoviesService {
+  constructor(private httpClient: HttpClient) {}
+  getData(category: string) {
```

Comment:
improve naming, getData is too anonymous, try to find something more specific to the actual request